src/components/ProfileNavigation/profileNavigation.styles.ts

```diff
@@ -0,0 +1,43 @@
+import { styled } from '@mui/system';
+import { Drawer, List, Toolbar } from '@mui/material';
+
+export const CustomToolbar = styled(Toolbar)(() => ({
+  justifyContent: 'flex-end',
+}));
+
+export const CustomDrawer = styled(Drawer)(({ theme }) => ({
+  [theme.breakpoints.down('md')]: {
+    '.MuiDrawer-paper': {
+      width: '100%',
+    },
+  },
+
+  '.MuiDrawer-paper': {
+    border: 0,
+  },
+}));
+
+export const CustomList = styled(List)(({ theme }) => ({
+  [theme.breakpoints.up('md')]: {
+    paddingTop: 70,
```

Comment:
You should use a spacing system, instead of a raw value.
https://mui.com/customization/spacing/

Reply:
Also, I think that `paddingTop` isn't necessary. A navigation list can be placed at the top of its container.